src/framework.cpp

```diff
@@ -630,8 +639,8 @@ PlutoConstraints *get_feautrier_schedule_constraints(PlutoProg *prog,
       continue;
     }
 
-    if (!pluto_stmt_is_member_of(dep->src, stmts, nstmts) ||
-        !pluto_stmt_is_member_of(dep->dest, stmts, nstmts)) {
+    if (!is_stmt_in_stmt_list(dep->src, stmts) ||
```

Comment:
Passing vector this way is all leading to unnecessary copies. Pass by const reference.